es6-classes/circle.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Circle */
+class Circle extends Shape {
+  constructor(area, perimeter, radius) {
+    super(area, perimeter);
```

Comment:
The only parameter for this `constructor` should be `radius`, and utilizing your `radius` you should be calculating what you'd want for your `area` & `perimeter` and pass those into `super()`